sys-devel/dev86/dev86-1.0.1.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit toolchain-funcs
+
+DESCRIPTION="Bruce's C compiler - Simple C compiler to generate 8086 code"
+HOMEPAGE="http://www.debath.co.uk/ https://github.com/lkundrak/dev86"
+SRC_URI="https://codeberg.org/jbruchon/dev86/archive/v1.0.1.tar.gz -> Dev86src-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+IUSE=""
+
+RDEPEND="sys-devel/bin86"
+DEPEND="${RDEPEND}
+	dev-util/gperf"
```

Comment:
gperf is only required for regenerating some headers that already bundled in tarball, so this DEPEND is rendundant.